### DISCUSSION TITLE
libstdc++-6.dll fails to load under Windows 11 with latest updates.

### DISCUSSION THREAD

**Discussion by theromorph** [setvisible/DownZemAll#204] [Q&A]
Hello.
After recent updates on Windows 11 got installed, I noticed, that DZA stopped working at all. Some excepts from event log follow:

Faulting application name: DownZemAll.exe, version: 0.0.0.0, time stamp: 0x647f7ebd
Faulting module name: libstdc++-6.dll, version: 0.0.0.0, time stamp: 0x00000000
Exception code: 0x40000015
Fault offset: 0x000000000001eeaa
Faulting process id: 0x0x3BC0
Faulting application start time: 0x0x1D9F43181825B58
Faulting application path: C:\Tools\DZA\DownZemAll.exe
Faulting module path: C:\Tools\DZA\libstdc++-6.dll
Report Id: 83b73ee2-d5e6-4afb-b480-53656f4b55d4
Faulting package full name: 
Faulting package-relative application ID: 

How coyld I fix this

**Comment by setvisible** [comment]
Hi,

Thanks for reporting. 

I don't have Windows 11 (yet), so I can't reproduce it... So you must be patient and wait for me to change my computer ;)

Or fix it yourself!

Regards,
